src/event/event-type.js

```diff
@@ -247,7 +247,19 @@ const CustomEventType: PKEventTypes = {
   /**
    * Fired when the drm license is responded from the DRM server
    */
-  DRM_LICENSE_LOADED: 'drmlicenseloaded'
+  DRM_LICENSE_LOADED: 'drmlicenseloaded',
+  /**
+   * Fired when tab state changes from in / out of focus
+   */
+  TAB_VISIBILITY_CHANGE: 'tabvisibilitychange',
+  /**
+   * Fired when the player gets in /out of scroll view
+   */
+  SCROLL_VISIBILITY_CHANGE: 'scrollvisibilitychange',
+  /**
+   * Fired when the player changes its visibility state - either by tabbing in / out or scrolling in /out of view
+   */
+  VISIBILITY_CHANGE: 'visibilitychange'
```

Comment:
I still don't see why we need to add seperate events, all say the same at the end of the day.
Either check with Kava if tab is same as scroll - i believe it does, or add the change to include a reason with tab/scroll - which I think is not really needed.

Reply:
For example the floating feature is only interested of scrolling in / out and not tabbing in/out.
I guess other features might be interested of just tabbing.
I don't see a reason why not giving all options open

Reply:
So let's use reason

Reply:
Not relevant anymore. Code moved to Kaltura. Only new event will be dispatched from Kaltura - VISIBILITY_CHANGE